integration-tests/camel-aws-s3/src/main/resources/application.properties

```diff
@@ -0,0 +1,14 @@
+quarkus.ssl.native=true
```

Comment:
The bad news is that SSL won't work on CI with native images.

@stuartwdouglas I asked a while ago why we did not download GraalVM directly instead of relying on Docker? The GitHub URL looks stable so we could automatize it. We download the Docker image anyway so I'm not sure it will generate more traffic.

Reply:
this is coming from the other integration test related to camel

Reply:
This seems to be needed, like the camel-salesforce integration test, removing it make the IT breaks